site/pages/events/index.js

```diff
@@ -1,15 +1,22 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
 import Container from '../../components/container';
 import Section from '../../components/section';
 import styles from './style.css';
 import EventsList from './events-list';
 import EventsBanner from '../../components/events-banner';
 
-export default function Events({ events }) {
+export default function Events({ events, eventsBanner }) {
+  const { url, altText, desktopURL, tabletURL, mobileURL } = eventsBanner[0];
```

Comment:
Possibly pedantic but it feels a bit odd to have `eventsBanner` as an array here (since the naming implies a singular instance). Instead we could deal with taking the first (and only) entry from the array in 'state/index.js'... up to you though, I don't feel strongly about it since it's only being used here.

Reply:
done